MatrixSDK/MXSession.h

```diff
@@ -517,6 +524,22 @@ typedef void (^MXOnBackgroundSyncFail)(NSError *error);
  */
 - (MXRoom *)privateOneToOneRoomWithUserId:(NSString*)userId;
 
+/**
+ The list of the direct rooms by user identifiers.
+ 
+ @return a dictionary where the keys are the user IDs and values are lists of room ID strings
+ of the 'direct' rooms for that user ID. nil if the direct rooms have not been yet fetched from the homeserver.
+ */
+@property (nonatomic, readonly) NSDictionary<NSString*, NSArray<NSString*>*> *directRooms;
+
+/**
+ Indicate if a room is a direct one.
+ 
+ @param roomId the id of the room.
+ @return YES if the room is direct.
+ */
+- (BOOL)isDirectRoom:(NSString*)roomId;
```

Comment:
Why not a isDirectRoom property in MXRoom like other room data (notificationCount, localUnreadEventCount, etc)?